src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -61,4 +61,9 @@
     <Compile Remove="corefx\System.Security.Cryptography\Perf.Rfc2898DeriveBytes.cs" />
   </ItemGroup>
 
+  <Target Name="BuildConfigurationCheck" BeforeTargets="Build">
```

Comment:
@jorive I added the warning

![obraz](https://user-images.githubusercontent.com/6011991/51954289-f694d780-243f-11e9-947e-dabf58f04e15.png)

I did not add two separate warnings because it's not possible to determine when the user did not provide the `--framework` and when it did in an explicit way `-f netcoreapp3.0`. I hope that this trade-off is acceptable.